pyproject.toml

```
@@ -0,0 +1,45 @@
+[build-system]
+build-backend = 'scikit_build_core.build'
+requires = [
+    'cmake>=3.25',
+    'numpy >= 1.23',
+    'scikit-build-core',
+    'wheel',
+]
+
+[project]
+dependencies = [
+  'numpy>=1.23',
+  'packaging>=20.0'
+]
+
+description = 'Serialbox - Serialization Library for C, C++, Fortran and Python'
+name = 'serialbox4py'
+# note: version should not be changed manually, use bump-my-version instead:
+# install: pip install bump-my-version
+# example: bump-my-version bump patch
+version = '2.6.2'
+license = {text = "BSD-3 License"}
+readme = {file = 'README.md', content-type = 'text/markdown'}
+
```

Suggestions for extra metadata:
```suggestion
authors = [{email = 'gridtools@cscs.ch'}, {name = 'ETH Zurich'}]
classifiers = [
  'Development Status :: 5 - Production/Stable',
  'Intended Audience :: Science/Research',
  'License :: OSI Approved :: BSD License',
  'Operating System :: POSIX',
  'Programming Language :: Python',
  'Programming Language :: Python :: 3',
  'Programming Language :: Python :: 3 :: Only',
  'Programming Language :: Python :: 3.8',
  'Programming Language :: Python :: 3.9',
  'Programming Language :: Python :: 3.10',
  'Programming Language :: Python :: 3.11',
  'Programming Language :: Python :: 3.12',
  'Programming Language :: Python :: Implementation :: CPython',
  'Topic :: File Formats',
  'Topic :: Scientific/Engineering :: Atmospheric Science',
  'Topic :: Scientific/Engineering :: Physics',
  'Topic :: Software Development :: Testing',
  'Topic :: System :: Archiving'
]

```